app/views/items/_item_grid.html.erb

```diff
@@ -0,0 +1,20 @@
+<div id="items">
+  <div class="container">
+      <div class="row ">
+        <% @items.each do |item| %>
+          <div class="col-sm-3 mb-4">
+          <div class="card">
+            <img class="card-img-top" src="https://via.placeholder.com/400x300" alt="Card image cap">
+            <div class="card-body text-center">
+              <h5 class="card-title text-truncate"><%= item.name %></h5>
+              <p class="card-text text-truncate">von: Alan Must</p>
+              <p class="card-text text-truncate text-success">verf&uuml;gbar</p>
+              <%= link_to "Show item", item, class: "card-link" %>
+              <%= link_to "Edit item", edit_item_path(item), class: "card-link" %>
```

Comment:
- i18n: please use consistently either English or German; I recognize that this might not be the case in other views, but do not introduce new discrepancies in views that are being created/modified in this MR.